### ISSUE TITLE
Too few arguments to function Laravel 5.5

### ISSUE THREAD

**Issue by xdisplayx** [cybercog/laravel-likeable#24]
Hello! To see if they could help me
In laravel 5.5 when I try to make a like or dislike (now I'm doing it massively with a seeder) the console returns this error

[Symfony\Component\Debug\Exception\FatalThrowableError]   Type error: Too few arguments to function Illuminate\Database\Eloquent\Model::setAttribute(), 1 passed in .....

Any idea why? Thank you!

**Comment by antonkomarev** [comment]
@xdisplayx  could you provide code samples, please?

**Comment by xdisplayx** [comment]
Yeah right!

```
//VotesTableSeeder

\App\Article::all()->each(function ($a) {
            \App\User::inRandomOrder()->take(14)->get()->each(function ($u) use ($a) {

                if (0 == (rand(0, 9) % 2)) {
                    $this->command->info('Creating Likes '. $u->id);
                    $a->like($u->id);
                } else {
                    $this->command->info('Creating Dislikes '. $u->id);
                    $a->dislike($u->id);
                }
            });
        });

```
If it proves, it fails `$a->like($u->id);` And `$a->dislike($u->id);`

``` 

//cmd

Seeding: UsersTableSeeder
Seeding: CategoriesTableSeeder
Seeding: ArticlesTableSeeder
Seeding: CommentsTableSeeder
Seeding: VotesTableSeeder
Creating Dislikes 5


  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Type error: Too few arguments to function Illuminate\Database\Eloquent\Model::setAttribute(), 1 passed in .....\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Concerns\HasAttributes.php on line 525
  and exactly 2 expected

``` 


Thank you!

**Comment by a-komarev** [comment]
I've got same error in other package. Will try to figure it out.

**Comment by antonkomarev** [comment]
Thats happens because of `const UPDATED_AT = null;` in `Like` model. Trying to figure out why this breaking change was introduced.

**Comment by xdisplayx** [comment]
Ok hopefully guide us in a solution thanks!

**Comment by antonkomarev** [comment]
@xdisplayx Issue fixed and published in release 3.0.1.

In 4.0.0 there will be changes in database and we'll add missing `updated_at` column and remove `setUpdatedAt` temporary added method which fixing an issue right now.

**Comment by xdisplayx** [comment]
@a-komarev  Updated and working, thank you very much!
And look forward to those great improvements, Regards!
